integration_tests/shell.nix

```diff
@@ -13,12 +13,12 @@ pkgs.mkShell {
 
     # tools
     cosmovisor
-    hermes
     solomachine
 
     # chain-maind for testing
     chain-maind-zemu
     chain-maind-zemu.instrumented
+    (import ../nix/hermes.nix { inherit pkgs; })
```

Comment:
I prefer to put hermes in the overlay(`nix/default.nix`), and just ref `pkgs.hermes` here.